internal/provider/access_policies_data_source.go

```diff
@@ -73,8 +73,50 @@ func (d *accessPoliciesDataSource) Schema(_ context.Context, _ datasource.Schema
 							ElementType: types.StringType,
 						},
 						"credential_provider": schema.StringAttribute{
-							Description: "Credential Provider ID configured in the Access Policy.",
+							Description:        "Configured server workload of the access policy.",
+							Computed:           true,
+							DeprecationMessage: "Deprecated",
+						},
+						"credential_providers": schema.ListNestedAttribute{
+							Description: "Set of Credential Providers to enforce on the Access Policy.",
+							Optional:    true,
 							Computed:    true,
+							NestedObject: schema.NestedAttributeObject{
+								Attributes: map[string]schema.Attribute{
+									"policy_id": schema.StringAttribute{
+										Description: "ID of access policy",
+										Optional:    true,
+									},
+									"credential_provider_id": schema.StringAttribute{
+										Description: "ID of credential provider.",
+										Required:    true,
+									},
+									"mapping_type": schema.StringAttribute{
+										Description: "Mapping type for the credential provider.",
+										Required:    true,
+									},
+									"header_name": schema.StringAttribute{
+										Description: "Name of the header for the credential provider.",
+										Optional:    true,
+									},
+									"header_value": schema.StringAttribute{
+										Description: "Value of the header for the credential provider.",
+										Optional:    true,
+									},
+									"httpbody_field_path": schema.StringAttribute{
+										Description: "Field path in the HTTP body for the credential provider.",
+										Optional:    true,
+									},
+									"httpbody_field_value": schema.StringAttribute{
+										Description: "Field value in the HTTP body for the credential provider.",
+										Optional:    true,
+									},
+									"account_name": schema.StringAttribute{
+										Description: "Name of the snowflake account for the credential provider.",
```

Comment:
Is this field only for snowflake? And can we capitalize snowflake -> Snowflake?

Reply:
As far as I know @quantifex ?